### PR TITLE
chore: improve pr template

### LINKED ISSUE #26561: PR template improvement
Patak recently [posted about improving the PR template of the vite repo](https://elk.zone/m.webtoo.ls/@patak/112155977618264831 ) and merged the improvements in https://github.com/vitejs/vite/pull/16266.

I agree with these two points in his PR description:

> - I don't think the "purpose of this pull request" section is useful. ... we already have this information in the PR title as we use conventional commit (fix, feat, docs, chore, etc).
> - Nobody is completing the "Before submitting the PR" section. The checkboxes doesn't render so it isn't good UX for contributors.

What do you think about changing this in the Nuxt repo as well?

### PR DESCRIPTION
**Pull request by Lehoczky**
### 🔗 Linked issue

resolves #26561

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have added tests (if possible).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,37 +1,20 @@
...
-### 📝 Checklist
+<!----------------------------------------------------------------------
+Before creating the pull request, please make sure you do the following:
 
-<!-- Put an `x` in all the boxes that apply. -->
-<!-- If your change requires a documentation PR, please link it appropriately -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- Check that there isn't already a PR that solves the problem the same way. If you find a duplicate, please help us reviewing it.
+- Read the contribution docs at https://nuxt.com/docs/community/contribution
+- Read the Contributing Guidelines at https://github.com/vitejs/vite/blob/main/CONTRIBUTING.md.
```
Comment (Lehoczky) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: These two could probably be combined into one page

**Comment by Lehoczky** [comment]
If this template is okay, I can go ahead today and create a PR for unjs repositories as well.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,37 +1,20 @@
-<!---
-☝️ PR title should follow conventional commits (https://conventionalcommits.org)
-
-Please carefully read the contribution docs before creating a pull request
- 👉 https://nuxt.com/docs/community/contribution
--->
-
 ### 🔗 Linked issue
 
 <!-- Please ensure there is an open issue and mention its number as #123 -->
```
Comment (danielroe) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
<!-- Please ensure there is an open issue and mention its number. For example, "resolves #123" -->
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (danielroe) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: but these are the vite contributing guidelines, right?

Comment (Lehoczky) [reply .github/PULL_REQUEST_TEMPLATE.md]: Oh yeah, my mind just went on autopilot and thought to link pointed to a `CONTRIBUTING.md` in this repository. Awkward🙃

**Review by danielroe — APPROVED** [review]
I'm more than happy to try this as an experiment and see how it is for people raising PRs...

Equally, if other maintainers have comments or feedback, I'd be glad to hear it.

**Pull request merged**